docs/release/trg-4/trg-4-06.md

```diff
@@ -148,9 +131,9 @@ You have to adapt some of the provided links to match your used version.
 - GitHub project: [https://github.com/dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker)
 - DockerHub: [https://hub.docker.com/_/microsoft-dotnet-aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet)
 
-### Linux
+### Hashicorp Vault image
 
-- Base image reference (example): `alpine:3.17`
-- Dockerfile: [alpinelinux/docker-alpine:3.17](https://github.com/alpinelinux/docker-alpine/blob/681b8c677aaed66e48a5ce721509647bd4dcd017/x86_64/Dockerfile)
-- GitHub project: [https://github.com/alpinelinux/docker-alpine](https://github.com/alpinelinux/docker-alpine)
-- DockerHub: [https://hub.docker.com/_/alpine](https://hub.docker.com/_/alpine)
+- Base image reference (example): `vault:13.3`
+- Dockerfile (:13.3): [vault:13.3](https://github.com/hashicorp/docker-vault/blob/v1.13.3/0.X/Dockerfile)
+- GitHub project: [https://github.com/hashicorp/docker-vault](https://github.com/hashicorp/docker-vault)
+- DockerHub: [hhttps://hub.docker.com/_/vault](https://hub.docker.com/_/vault)
```

Comment:
Please revert the change of replacing the linux alpine base image which the vault image

Reply:
That has to be a diff / move conflict. Will fix it

Reply:
fixed